lib/resolveurl/plugins/mixdrop.py

```diff
@@ -0,0 +1,42 @@
+"""
+    Kodi resolveurl plugin
+    Copyright (C) 2019
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import re
+from resolveurl import common
+from resolveurl.resolver import ResolveUrl, ResolverError
+from lib import jsunpack
+
+class MixdropResolver(ResolveUrl):
+    name = "mixdrop"
+    domains = ["mixdrop.co"]
+    pattern = '//(mixdrop\.co)/(?:f|e)/(.+)'
```

Comment:
Please stick to the standard pattern format, this handles in case there is an embed. or www. in front of mixdrop.co
`pattern = r'(?://|\.)(mixdrop\.co)/(?:f|e)/([0-9a-zA-Z]+)'`